css/main.css

```diff
@@ -274,7 +279,150 @@ nav {
 
 /* WHO WE ARE */
 .who-we-are {
-  height: 850px;
+  background: url(../images/who-we-are.png);
```

Comment:
NIT: Using background-image instead of just background is actually slightly more efficient...